qualtran/bloqs/comparison_gates.ipynb

```diff
@@ -38,14 +37,16 @@
    },
    "source": [
     "## Introduction\n",
-    "The current optimal implementation of a reversible oracle for comparing two quantum numbers of $n$ qubits each is $8n + \\mathcal{O}(1)$ and is given in the supplementary materials of [Berry et al., 2018](https://doi.org/10.1038/s41534-018-0071-5). Their implementation uses divide and conquer technique to create a binary tree of depth $\\log_2{(n)}$ whose leafs are the qubits of the numbers with intermediate values stored in $\\mathcal{O}(n)$ ancillas. Each non leaf node uses two CSWAP operations and the value of the comparison is computed from the root node using one Toffoli. \n",
+    "The current optimal implementation of a reversible oracle for comparing two quantum numbers of $n$ qubits each is $4n + \\mathcal{O}(1)$ T operations. This is done by reducing the problem to subtraction which takes $4n + \\mathcal{O}(1)$ (see. [Gidney., 2018](https://arxiv.org/abs/1709.06648)). \n",
```

Comment:
why is there a period and a comma in `Gidney.,`?

Reply:
done